HW_1/Ex_6/Ex_6.py

```diff
@@ -0,0 +1,28 @@
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы
+# расплачивались за проезд и получали билет с номером. Счастливым
+# билетом называют такой билет с шестизначным номером, где сумма
+# первых трех цифр равна сумме последних трех. Т.е. билет с номером
+# 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать
+# программу, которая проверяет счастливость билета
+
+# 123 456
+
+# ___________________РЕШЕНИЕ_________________________
+
+num = input("Введите шестизначный номер билета: ")
+if len(num) == 6:
+    leftNum = int(num) // 1000
```

Comment:
Кстати, в Python принято (рекомендуется) использовать `snake_case`, вместо `CamelCase`. То есть в вашем случае можно было сделать так
```suggestion
    left_num = int(num) // 1000
```